packages/p2p/src/components/floating-rate/floating-rate.scss

```diff
@@ -0,0 +1,148 @@
+.floating-rate {
+    display: flex;
+    flex-direction: column;
+    &__field {
+        display: flex;
+    }
+    &__input {
+        height: 40px;
+        align-self: center;
+        appearance: none;
+        box-sizing: border-box;
+        border-radius: $BORDER_RADIUS;
+        color: var(--text-general);
+        border: 1px solid var(--border-normal);
+        background-image: none;
+        input::-webkit-input-placeholder {
+            /* WebKit, Blink, Edge */
+            color: var(--text-less-prominent) !important;
+        }
+        input:-moz-placeholder {
+            /* Mozilla Firefox 4 to 18 */
+            color: var(--text-less-prominent) !important;
+            opacity: 1;
+        }
+        input::-moz-placeholder {
+            /* Mozilla Firefox 19+ */
+            color: var(--text-less-prominent) !important;
+            opacity: 1;
+        }
+        input:-ms-input-placeholder {
+            /* Internet Explorer 10-11 */
+            color: var(--text-less-prominent) !important;
+        }
+        input::-ms-input-placeholder {
+            /* Microsoft Edge */
+            color: var(--text-less-prominent) !important;
+        }
+
+        &:hover {
+            border-color: var(--border-hover);
+        }
+        &:active,
+        &:focus {
+            border-color: var(--border-active);
+        }
+
+        &__error_field {
+            color: $color-red-1;
+            border-color: $color-red;
+        }
+    }
+
+    &__percent {
+        position: absolute;
+        height: 3.2rem;
+        right: 4rem;
+        align-items: center;
+        justify-content: center;
+        display: flex;
+        background: transparent;
+        border-color: transparent;
+        z-index: 2;
+        color: inherit;
+
+        &--symbol {
+            font-size: 1.4rem;
+            line-height: 1.5;
+            padding-top: 0.7rem;
+        }
+        &:before {
+            @include typeface(--paragraph-center-normal-black);
+            color: inherit;
+        }
+    }
+
+    &__mkt-rate {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+        padding: 2px 8px;
```

Comment:
Please use `rem` since `px` is not scalable.

Reply:
Will make the change